dist/gaia-header.js

```diff
@@ -14,39 +11,25 @@ var noop  = function() {};
  * @return {Boolean}
  */
 var hasShadowCSS = (function() {
-  var div = document.createElement('div');
-  try { div.querySelector(':host'); return true; }
+  try { document.querySelector(':host'); return true; }
```

Comment:
Looks like perhaps you built without updating your local version of `gaia-component`?